lib/browser/api/web-contents.ts

```diff
@@ -522,17 +523,17 @@ WebContents.prototype._callWindowOpenHandler = function (event: Electron.Event,
     event.preventDefault();
     return defaultResponse;
   } else if (response.action === 'allow') {
+    defaultResponse.outlivesOpener = typeof response.outlivesOpener === 'boolean' ? response.outlivesOpener : false;
```

Comment:
It seems a little odd to be modifying `defaultResponse` here. Can we perhaps name that to `finalResponse`, or otherwise refactor this to avoid modifying `defaultResponse`?